uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutView.kt

```diff
@@ -341,6 +341,7 @@ internal class CheckoutView @JvmOverloads constructor(context: Context,
                                                            .COUNTRY_DIALING_CODE_KEY) ?: ""
 
             passengersDetailLayout.setCountryFlag(countryCode, dialingCode, true)
+            passengersDetailLayout.setFocusOnPhoneNumber()
```

Comment:
How about we do the `mobileNumberInput.requestFocus()` inside the `setCountryFlag` so we would avoid adding a new method to PassengerDetailsContract

Reply:
You can even add a new parameter to    `setCountryFlag(countryCode: String, dialingCode: String, validateField: Boolean, focus: Boolean)` to avoid unnecessary focus